src/lv_demo_benchmark/lv_demo_benchmark.c

```diff
@@ -685,14 +685,16 @@ static void scene_next_task_cb(lv_timer_t * timer)
             scene_act ++;
         }
         opa_mode = false;
-    } else {
+    }
+    else {
         if(scenes[scene_act].time_sum_normal == 0) scenes[scene_act].time_sum_normal = 1;
         scenes[scene_act].fps_normal = (1000 * scenes[scene_act].refr_cnt_normal) / scenes[scene_act].time_sum_normal;
+        scene_act ++;
```

Comment:
`scene_act` is already incremented [here](https://github.com/lvgl/lv_demos/pull/118/files#diff-4ddf7d85c3bb1076e03e92e8339b4a0c031d9bf93207e1c8592331976f206a14R685).

We should go to the next scene if both the normal and opa mode is ready. 


Maybe I missed the point of the fix, please provide a short description what is the issue you have found.